tests/testthat/test-read_graph.R

```diff
@@ -6,3 +6,17 @@ test_that("reading GraphML file works", {
 
   expect_true(isomorphic(g2, g))
 })
+
+test_that("reading graph in NCOL format", {
+  ncol_path <- tempfile("testfile", fileext = ".ncol")
+  g <- make_graph(c(1, 2, 2, 3))
+  write_graph(g, ncol_path, "ncol")
+  expect_no_error(g2 <- read_graph(ncol_path, "ncol"))
```

Comment:
do we want to know more than no error?

Reply:
> do we want to know more than no error?

I supposed g and g2 are identical at the beginning, but they are not. The graphs have the same structures but there's some difference in properties. So I replaced `expect_true(identical_graphs(g, g2))` with `expect_no_error` to meet the minimum requirement. If that's not expected, there should be other changes after thorough inspection.